### PR TITLE
DPLT-102: create a new theme

### PR DESCRIPTION
**Pull request by paulbanjawan**
Link to exercise : https://drupalize.me/tutorial/exercise-create-new-theme?p=3291

Updates: 
- Created a new custom theme called reboot

### CODE REVIEW

File: web/themes/custom/icecream/icecream.info.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# THEMENAME.info.yml file for Ice Cream example theme.
+name: Ice Cream
```
Comment (xianvalencia) [inline web/themes/custom/icecream/icecream.info.yml]: @paulbanjawan We only need one theme for this training, you created two. The one below is more adherent to the exercise than this one. You can delete this file.

**Review by xianvalencia — CHANGES_REQUESTED** [review]
@paulbanjawan Added a comment on this PR. And out of nowhere, can you explain why you named your theme "Reboot"? There's no right/wrong answer here. 😁

File: web/themes/custom/icecream/icecream.info.yml (hunk 1 above)
Comment (paulbanjawan) [reply web/themes/custom/icecream/icecream.info.yml]: got it @xianvalencia , will delete the icecream theme

**Review by xianvalencia — COMMENTED** [review]
Looks good.

**Review by xianvalencia — APPROVED** [review]

**Pull request merged**